package/yast2-schema-micro.spec

```diff
@@ -0,0 +1,112 @@
+#
+# spec file for package yast2-schema
+#
+# Copyright (c) 2013 SUSE LINUX Products GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+
+Name:           yast2-schema-micro
+Version:        4.4.9
+Release:        0
+
+BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+Source0:        yast2-schema-%{version}.tar.bz2
+
+Group:	        System/YaST
+License:        GPL-2.0-or-later
+
+Url:            https://github.com/yast/yast-schema
+
+# provide schema, so it is full replacement
+Provides:       yast2-schema
+# both cannot be used as it uses same files
+Conflicts       yast2-schema
```

Comment:
The colon symbol is missing here.

Reply:
ah, good catch, I found it during testing and forget to submit.